Khan/perseus
# [Locked Figure Labels] View locked ellipse labels

## Summary:
- Add feature flag for locked ellipse labels
- Add labels to LockedEllipseType
- Update builder
- Update stories and tests

Issue: https://khanacademy.atlassian.net/browse/LEMS-2349

## Test plan:
- Go to http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--mafs-with-locked-ellipse-labels-flag
- Confirm that the visible label is on the locked ellipse only
- Go to the point, line, and vector  stories and confirm that the
  locked ellipse does not have a visible label (make sure the
  flags are separating the behaviors correctly)

<img width="444" alt="image" src="https://github.com/user-attachments/assets/dedc34d4-8cad-455a-b281-4c987f732ef4">
